samples/nrf9160/serial_lte_modem/src/slm_at_host.c

```diff
@@ -346,3 +359,36 @@ int slm_at_host_init(void)
 	LOG_DBG("at_host init done");
 	return err;
 }
+
+int slm_at_host_uninit(void)
```

Comment:
This function could be `void`, since we do a best-effort to un-initialize everything and keep on going until we enter sleep mode.

Reply:
Will update